internal/services/cdn/cdn_frontdoor_firewall_policy_resource_test.go

```diff
@@ -333,6 +334,7 @@ resource "azurerm_cdn_frontdoor_firewall_policy" "test" {
   custom_rule {
     name                           = "Rule1"
     enabled                        = true
+    request_body_check_enabled     = false
```

Comment:
I think this is in the wrong place? it should be pulled up into the root level of the resource?

```
------- Stdout: -------
=== RUN   TestAccCdnFrontDoorFirewallPolicy_update
=== PAUSE TestAccCdnFrontDoorFirewallPolicy_update
=== CONT  TestAccCdnFrontDoorFirewallPolicy_update
    testcase.go:113: Step 1/6 error: Error running pre-apply refresh: exit status 1
        
        Error: Unsupported argument
        
          on terraform_plugin_test.tf line 47, in resource "azurerm_cdn_frontdoor_firewall_policy" "test":
          47:     request_body_check_enabled     = false
        
        An argument named "request_body_check_enabled" is not expected here.
--- FAIL: TestAccCdnFrontDoorFirewallPolicy_update (13.25s)
FAIL
```